test/browser/time-picker/time-picker-spec.js

```diff
@@ -56,7 +62,10 @@ describe('TimePicker', () => {
     );
 
     const timeTextField = TestUtils.findRenderedComponentWithType(render, TextField);
+    const timepicker = TestUtils.findRenderedComponentWithType(render, TimePicker);
 
-    expect(timeTextField.props.value, formatTime(valueTime));
+    expect(formatTime(timepicker.state.time)).to.be.equal(timeTextField.props.value);
```

Comment:
Can we put all the assertions in these tests in the same "order" in terms of what we're checking -- get's confusing.